src/utils/check-utils.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Checks whether an object has any null or undefined props
+ *
+ * @param object Object to be null-checked on every of its prop
+ * @returns boolean, indicates if any prop is null within the object
+ */
+export const hasAllPropsDefined = (object: object): boolean => {
```

Comment:
This works. Although rather than using this there might be some build in form validation functionality with the Mui form control which could be a good option.

Reply:
I can set the form fields as required.